src/pdf/DriverOverview/DriverOverviewPdfButton.tsx

```diff
@@ -94,32 +123,56 @@ type DriverPdfResponse =
+    parcel: ParcelForDelivery
+): DriverOverviewRowData => {
+    const client = parcel.client;
+    const clientIsActive = parcel.client.is_active;
+    return {
+        name: clientIsActive ? client?.full_name ?? "" : displayNameForDeletedClient,
+        address: {
+            line1: client?.address_1 ?? "",
+            line2: client?.address_2 ?? null,
+            town: client?.address_town ?? null,
+            county: client?.address_county ?? null,
+            postcode: client?.address_postcode,
+        },
+        contact: clientIsActive ? client?.phone_number ?? "" : "-",
+        packingDate: formatDateToDate(parcel.packing_date) ?? null,
+        instructions: clientIsActive ? client?.delivery_instructions ?? "" : "-",
+        clientIsActive: clientIsActive,
+        numberOfLabels: parcel.labelCount,
+        collectionCentre: parcel.collection_centre?.name,
+        isDelivery: parcel.collection_centre?.is_delivery,
+    };
+};
+
+const transformParcelDataToClientInformation = (
+    parcels: ParcelForDelivery[]
+): DriverOverviewTablesData => {
+    const collections: DriverOverviewRowData[] = [];
+    const deliveries: DriverOverviewRowData[] = [];
+
+    for (const parcel of parcels) {
+        if (parcel.collection_centre?.is_delivery) {
+            deliveries.push(transformRowToDriverOverviewTableData(parcel));
+        } else {
+            collections.push(transformRowToDriverOverviewTableData(parcel));
+        }
+    }
+
+    return {
+        collections: collections.sort(compareCollectionCentres),
+        deliveries: deliveries.sort(comparePostcodes),
+    };
+};
+
 const getDriverPdfData = async (parcelIds: string[]): Promise<DriverPdfResponse> => {
-    const clientInformation = [];
     const { data: parcels, error: parcelsError } = await getParcelsForDelivery(parcelIds);
     if (parcelsError) {
         return { data: null, error: parcelsError };
     }
-    for (const parcel of parcels) {
-        const client = parcel.client;
-        const clientIsActive = parcel.client.is_active;
-        clientInformation.push({
-            name: clientIsActive ? client?.full_name ?? "" : displayNameForDeletedClient,
-            address: {
-                line1: client?.address_1 ?? "",
-                line2: client?.address_2 ?? null,
-                town: client?.address_town ?? null,
-                county: client?.address_county ?? null,
-                postcode: client?.address_postcode,
-            },
-            contact: clientIsActive ? client?.phone_number ?? "" : "-",
-            packingDate: formatDateToDate(parcel.packing_date) ?? null,
-            instructions: clientIsActive ? client?.delivery_instructions ?? "" : "-",
-            clientIsActive: clientIsActive,
-            numberOfLabels: parcel.labelCount,
-        });
-    }
-    clientInformation.sort(compareDriverOverviewTableData);
+    const clientInformation = transformParcelDataToClientInformation(parcels);
```

Comment:
Are clientInformation and transformParcelDataToClientInformation accurate names here? They looks like the transform parcel data to me

Reply:
Renamed in latest :)